Gemfile

```diff
@@ -1,3 +1,78 @@
+# Versioning Policy
+# =================
+#
+# In most cases gems should be added with a pessimistic constraint to their
+# PATCH level:
+#
+#    gem 'foo', '~> 1.2.0'
+#
+# This will install the latest version of the 1.2 series. When 1.3 or 2.0 are
+# released, we can assume there may be a breaking change.
+#
+# If we need to be more specific than this, apply one of the following
+# guidelines. Use commit messages to describe why the extra specificity is
+# required. Links to changelogs are very appreciated.
+#
+# If we need greater than a specific PATCH level, specify that:
+#
+#    gem 'foo', '~> 1.2.7'
```

Comment:
I'm wondering whether this should be

```ruby
gem 'foo', '~> 1.2.0', '>= 1.2.7'
```

Its the same, but it does make it more explicit that we really do need `1.2.7+`